indexing-service/src/main/java/org/apache/druid/indexing/common/index/RealtimeAppenderatorTuningConfig.java

```diff
@@ -93,6 +96,7 @@ public RealtimeAppenderatorTuningConfig(
       @JsonProperty("maxSavedParseExceptions") @Nullable Integer maxSavedParseExceptions
   )
   {
+    this.appendableIndexSpec = appendableIndexSpec == null ? DEFAULT_APPENDABLE_INDEX : appendableIndexSpec;
```

Comment:
I agree. But that will require a refactor to unify some of the `TuningConfig` implementations to one common (abstract?) implementation.
This can be done before or after this PR.